CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 1.11.0-beta.1
+
+### ✨ Features
+* Add scale option to markers, to support custom sizing when using the default Marker SVG element.([#9414](https://github.com/mapbox/mapbox-gl-js/pull/9414)) (h/t [adrianababakanian](https://github.com/adrianababakanian))
+
+### 🍏 Improvements
+* Improving the shader compilation speed by manually getting the run-time attributes and uniforms.([#9497](https://github.com/mapbox/mapbox-gl-js/pull/9497))
+* Mapbox gl-js documentation improvements.([#9607](https://github.com/mapbox/mapbox-gl-js/pull/9607))
```

Comment:
Same here.

Reply:
can't find other instances of `9607` either..

Reply:
I don't think we mentioned the docs improvement as part of the release changelog, that's probably why it's not in there